### PR TITLE
Modularize code for #36. 

### PR DESCRIPTION
**Pull request by thakkardharmik**
Please review the code. I need some help to modify setup.py in case the code looks fine. Also if someone can help me test the code.

### CODE REVIEW

File: Listener.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import tweepy
+import os
+import csv
+import json
+import sys
+from os.path import expanduser
+from config.ConfigReader import ConfigurationReader
```
Comment (0x006E) [inline Listener.py]: This is not needed I think

File: Listener.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import csv
+import json
+import sys
+from os.path import expanduser
+from config.ConfigReader import ConfigurationReader
+from importlib import reload
+
+# Twitter API credentials
+cfg = {}
+home = expanduser("~")
```
Comment (0x006E) [inline Listener.py]: These lines are not needed in this file

File: main.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
+from Twweeter import Twweeter
+import tweepy
+import os
+import csv
+import json
+import sys
+from os.path import expanduser
+from config.ConfigReader import ConfigurationReader
```
Comment (0x006E) [inline main.py]: Nt needed

File: main.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+from importlib import reload
+from Listener import Listener
+
+# Twitter API credentials
+home = expanduser("~")
+Configuration = ConfigurationReader()
```
Comment (0x006E) [inline main.py]: Nt needed.

Comment (0x006E) [inline main.py]: Not needed

**Review by 0x006E — APPROVED** [review]
Please check comments. I think the code will be alright. I didn't test it. But it should work perfectly though.

**Comment by thakkardharmik** [comment]
Implemented changes as suggested in comments.

**Comment by 0x006E** [comment]
@thakkardharmik What changes should be made in the setup.py?

**Comment by thakkardharmik** [comment]
@nithinswarrier I think some changes would be required @ https://github.com/CruiseDevice/twweet-cli/blob/master/setup.py#L97 and 
https://github.com/CruiseDevice/twweet-cli/blob/master/setup.py#L100

**Comment by CruiseDevice** [comment]
@thakkardharmik merged :+1: :sparkles: :tada:  Sorry for merging this so late. I was busy with other commitments.

**Pull request merged**